sentry-samples/sentry-samples-android/build.gradle.kts

```diff
@@ -88,6 +90,18 @@ android {
     kotlinOptions {
         jvmTarget = JavaVersion.VERSION_1_8.toString()
     }
+
+//    testOptions {
+//        execution = "ANDROIDX_TEST_ORCHESTRATOR"
+//    }
+}
+
+task<Wrapper>("wrapper") {
+    gradleVersion = "6.8.3"
+}
```

Comment:
we don't need this, we have https://github.com/getsentry/sentry-java/tree/main/gradle/wrapper